### PR TITLE
Add member <UserItem> to print cigroups|cigroup-members

### PR DESCRIPTION
**Pull request by taers232c**
Drop Google-Coordinate product ID
Documentation cleanup

### CODE REVIEW

File: src/gam/gapi/__init__.py [hunk 1]
```diff
@@ -218,6 +218,60 @@ def got_total_items_first_last_msg(items):
     return f'Got {TOTAL_ITEMS_MARKER} {items}: {FIRST_ITEM_MARKER} - {LAST_ITEM_MARKER}' + '\n'
 
 
+def process_all_pages_result(page, items, all_items, total_items, page_message, message_attribute):
```
Comment (jay0lee) [inline src/gam/gapi/__init__.py:221]: what are we gaining by pulling this into a function? Also the name vs description is confusing, are we processing all pages or one page here?

File: src/gam/gapi/cloudidentity/groups.py [hunk 2]
```diff
@@ -222,16 +229,30 @@ def print_():
                 display.add_titles_to_csv_file([
                     'Owners',
                 ], titles)
-    gam.printGettingAllItems('Groups', None)
+    gam.printGettingAllItems('Groups', usemember)
     page_message = gapi.got_total_items_first_last_msg('Groups')
-    entityList = gapi.get_all_pages(ci.groups(),
-                                    'list',
-                                    'groups',
+    if usemember:
```
Comment (jay0lee) [inline src/gam/gapi/cloudidentity/groups.py:234]: the challenge here is searchTransitiveGroups returns GroupRelation:

https://cloud.google.com/identity/docs/reference/rest/v1beta1/groups.memberships/searchTransitiveGroups#response-body

while groups.list just returns Groups:
https://cloud.google.com/identity/docs/reference/rest/v1beta1/groups/list#response-body

these two have some differences today but if that difference grows we end up with a whole bunch of if/then/else logic on this and it's just messy. searchTransitiveMemberships is similar.

I think it'd be cleaner to break these transitive API calls out into their own funciton (and thus own GAM command). The functions will be cleaner and easier to read and it means admins need to call another GAM command to use the transitive functions. Considering that the transitive functions only work on the Enterprise SKUs this makes sense also that some GAM commands won't work (as opposed to some arguments on some GAM commands not working).

**Comment by taers232c** [comment]
I put the usemember code back
It's purpose is to select groups to print, none of the data it returns is printed.
It's use is just like print groups
```
gam print cigroups member user@domain.com
gam print groups member user@domain.com
```

**Comment by taers232c** [comment]
Will re-issue for 5.24

**Pull request closed without merging**